### ISSUE TITLE
The text color is not consistent across the website/Accessibility

### ISSUE THREAD

**Issue by Fw7424** [wdetdigital/wdet-conversion#158]
The paragraph text across the site is not consistent. Also fails accessibility for contrast in many cases. 

![Screen Shot 2022-04-21 at 11 57 23 AM](https://user-images.githubusercontent.com/41925609/164502395-bacf19cc-0885-426d-8112-78ad639bc0be.png)
![Screen Shot 2022-04-21 at 11 59 03 AM](https://user-images.githubusercontent.com/41925609/164502400-42b8fc6e-58c7-4f37-8b94-79df5bff68d1.png)
![Screen Shot 2022-04-21 at 11 57 49 AM](https://user-images.githubusercontent.com/41925609/164502403-2280eb22-dba0-42b0-833e-ace066a059cd.png)
![Screen Shot 2022-04-21 at 11 57 38 AM](https://user-images.githubusercontent.com/41925609/164502406-aff7b902-004e-4dd4-bf95-c435c69a119e.png)

**Comment by Fw7424** [comment]
This change can be made via theme pages. this doesn't need to be done first on development.
